src/coreclr/vm/interpexec.cpp

```diff
@@ -1856,8 +1856,9 @@ void InterpExecMethod(InterpreterFrame *pInterpreterFrame, InterpMethodContextFr
                     methodSlot = ip[3];
                     int32_t targetAddrSlot = ip[4];
                     int32_t indirectFlag = ip[5];
+                    bool suppressGCTransition = (ip[6] != 0);
```

Comment:
nit: I think we could merge this together with indirectFlag, to reduce code size